src/module.c

```diff
@@ -842,6 +842,8 @@ JL_DLLEXPORT jl_module_t *jl_get_module_of_binding(jl_module_t *m, jl_sym_t *var
 
 static NOINLINE void print_backdate_admonition(jl_binding_t *b) JL_NOTSAFEPOINT
 {
+    if (jl_options.depwarn == JL_OPTIONS_DEPWARN_ERROR)
+        jl_undefined_var_error(b->globalref->name, (jl_value_t*)b->globalref->mod);
     jl_safe_printf(
```

Comment:
Should maybe mention `--depwarn=error` in this warning so that people know how to turn it on

Reply:
Good point. I'll steal the message from Vararg{T} where T